diff --git a/node_modules/ag-grid-vue3/lib/VueFrameworkComponentWrapper.js b/node_modules/ag-grid-vue3/lib/VueFrameworkComponentWrapper.js
index a142008..8370beb 100644
--- a/node_modules/ag-grid-vue3/lib/VueFrameworkComponentWrapper.js
+++ b/node_modules/ag-grid-vue3/lib/VueFrameworkComponentWrapper.js
@@ -17,12 +17,21 @@ export class VueFrameworkComponentWrapper extends BaseComponentWrapper {
                 super.init(params);
             }
             hasMethod(name) {
-                return wrapper.getFrameworkComponentInstance()[name] != null;
+                const componentInstance = wrapper.getFrameworkComponentInstance()
+                if (!componentInstance[name]) {
+                    return componentInstance?._?.exposed?.[name] !== null
+                } else {
+                    return true
+                }
             }
             callMethod(name, args) {
                 const componentInstance = this.getFrameworkComponentInstance();
                 const frameworkComponentInstance = wrapper.getFrameworkComponentInstance();
-                return frameworkComponentInstance[name].apply(componentInstance, args);
+                if (frameworkComponentInstance[name]) {
+                    return frameworkComponentInstance[name].apply(componentInstance, args);
+                } else {
+                    return frameworkComponentInstance?._?.exposed?.[name]?.apply(componentInstance, args);
+                }
             }
             addMethod(name, callback) {
                 wrapper[name] = callback;
